scripts/versions.py

```diff
@@ -152,8 +184,7 @@ def get_root_dir():
         root_dir = subprocess.check_output(["git", "rev-parse", "--show-toplevel"]).decode().strip()
         return root_dir
     except subprocess.CalledProcessError:
-        print("ERROR: Not a git repository or unable to determine root directory.")
-        sys.exit(1)
+        error_exit(title="Git repository not found", message="Not a git repository or unable to determine root directory.")
```

Comment:
Do we need title and message both for these logs?
I think we should still keep error and exit separate or structure where exit function takes in error if any.

log(AnnotationType, msg) => usage => log( ERROR, "Not a git repository or unable to determine root directory")

so we could log warnings and debug messages as mentioned in other comments.